modules/crio_setconfig.adoc

```diff
@@ -0,0 +1,132 @@
+you should only change CRI-O configuration files through {product-title} or for the
+purposes of testing or troubleshooting CRI-O. On a running {product-title} node, most
+CRI-O configuration settings are kept in the `/etc/crio/crio.conf` file.
+
+Settings in a `crio.conf` file define how storage, the listening socket,
+runtime features, and networking are configured for CRI-O.
+Here's an example of the default `crio.conf` file (look in the file itself
+to see comments describing these settings):
+
+```
+[crio]
+root = "/var/lib/containers/storage"
+runroot = "/var/run/containers/storage"
+storage_driver = "overlay"
+storage_option = [
+        "overlay.override_kernel_check=1",
+   ]
+
+[crio.api]
+listen = "/var/run/crio/crio.sock"
+stream_address = ""
+stream_port = "10010"
+file_locking = true
+
+[crio.runtime]
+runtime = "/usr/bin/runc"
+runtime_untrusted_workload = ""
+default_workload_trust = "trusted"
+no_pivot = false
+conmon = "/usr/libexec/crio/conmon"
+conmon_env = [
+        "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+   ]
+selinux = true
+seccomp_profile = "/etc/crio/seccomp.json"
+apparmor_profile = "crio-default"
+cgroup_manager = "systemd"
+hooks_dir_path = "/usr/share/containers/oci/hooks.d"
+default_mounts = [
+        "/usr/share/rhel/secrets:/run/secrets",
+   ]
+pids_limit = 1024
+enable_shared_pid_namespace = false
+log_size_max = 52428800
+
+[crio.image]
+default_transport = "docker://"
+pause_image = "docker.io/openshift/origin-pod:v3.11"
+pause_command = "/usr/bin/pod"
+signature_policy = ""
+image_volumes = "mkdir"
+insecure_registries = [
+""
+]
+registries = [
+"docker.io"
+]
+
+[crio.network]
+network_dir = "/etc/cni/net.d/"
+plugin_dir = "/opt/cni/bin"
+```
+
+The following sections describe how different CRI-O
+configurations might be used in the `crio.conf` file.
+
+== Configuring CRI-O storage
+OverlayFS2 is the recommended (and default) storage driver for {product-title},
+whether you use CRI-O or Docker as your container engine. See link:https://docs.openshift.com/container-platform/3.10/scaling_performance/optimizing_storage.html#choosing-a-graph-driver[Choosing a graph driver]
+for details on available storage devices.
+
+[NOTE]
+====
+Although devicemapper is a supported storage facility for CRI-O, the CRI-O
+garbage collection feature does not yet
+work with devicemapper and so is not recommended for production use. Also,
+see link:https://bugzilla.redhat.com/show_bug.cgi?id=1625394[BZ1625394] and
+link:https://bugzilla.redhat.com/show_bug.cgi?id=1623944[BZ1623944] for other
+devicemapper issues that apply to how both CRI-O and `podman` use
+container storage.
+====
+
+Things you should know about CRI-O storage include the facts that CRI-O storage:
+
+* Holds images by storing the root filesystem of
+each container, along with any layers that go with it.
+
+* Incorporates the same storage layer that is used with the docker service.
+
```

Comment:
s/docker service/Docker service or *docker* service. (Docker service is more common in our docs)

Reply:
It's literally the "docker" server. As in "systemctl start docker". But I'm speaking about it generically here, so I'll change it to "Docker" instead.